net/iputils/DEPENDS

```diff
@@ -1,3 +1,8 @@
 depends libcap
 depends openssl
 depends sysfsutils
+depends git
+
+if [ "`get_module_config DOCS`" = "y" ]; then
```

Comment:
No logic in the DEPENDS files. Make it an optional_depends and add "if in_depends $MODULE docbook-utils; then ... fi" in BUILD instead.